editgraph

```diff
@@ -1,14 +1,9 @@
 #!/bin/bash
-which open
-if [ $? == 0 ] 
-then
-   open https://controlcore-project.github.io/DHGWorkflow/
-else
```

Comment:
Thanks for your attempt at fixing this issue.

A few pointers:
1. When you make a fix to a bug, always make sure to do the minimal needed changes. So in this case, changing the URL was all you had to do. Other changes are irrelevant.
2. I have fixed these in the recent commits.
3. However, the actual reason for these bugs is due to the dev branch being stale. I am attempting to merge the main branch back into the dev. But it has some merge conflicts. So I will come back to that later. https://github.com/ControlCore-Project/concore/pull/28

In the meantime, please work on the main branch unless we explicitly ask you to work on the dev branch.

I close this pull request now.

Reply:
Thanks @pradeeban for your pointers. I'll consider your points. 

But I working on **main branch only**, this issue occur in main branch as well as in dev branch. I put pull requests in dev branch because in the documentation you mentioned "Please make sure to send your concore pull requests to the dev branch."

Let me tell you the problem in detail. Please follow my screenshots that I attached in pull requests.

- "Which open" command return 0 in Linux. So, first if condition satisfy and it use "open -a 'Google Chrome'  https://controlcore-project.github.io/concore-editor/
- But it will not work in Linux instead in Linux "xdg-open https://controlcore-project.github.io/concore-editor/" works.

Reply:
Hi @shivangvijay, thanks for the explanation. That makes sense.

The dev branch had become stale, and any commits to that were deviating from the main. Now I have merged #28. So you should be able to create pull requests there, as you pointed out from the README.

We were testing on Windows and Mac and apparently did not realize the issue you pointed out. Now I have adopted your changes and committed it. Please see the updates in the relevant bug report #25.